CONTRIBUTING.md

```diff
@@ -319,6 +319,6 @@ You want to keep an eye on Circle CI for the progress of the release ([0.3.1 exa
 
 ### Latest release
 
-To get the latest release you can use the link [https://github.com/weaveworks/eksctl/releases/latest]().
+To get the latest release you can use the link [https://github.com/weaveworks/eksctl/releases/latest](https://github.com/weaveworks/eksctl/releases/latest).
```

Comment:
```suggestion
To get the latest release you can use the link <https://github.com/weaveworks/eksctl/releases/latest>.
```

Reply:
Thanks a lot for fixing the link. This might be a more compact way of writing it.

Reply:
Oh thanks, I didn't know about this < > :+1:

Reply:
@dholbach I noticed that you have something like link checker in flagger, any plan to bring those checks to eksctl as well ?

Reply:
Yes. So we have link checkers for `flux` and `wksctl` and are going to land them for `ignite` and `helm-operator` too. Sounds like a good idea!

Reply:
:100: 

I mentioned flagger wrong haha, i noticed about link checker as one of my pr in flux failed :+1:,  then I took a look and found out you have implemented very extensive link checkers :100: :medal_military: